src/scripts/install.py

```diff
@@ -140,11 +144,20 @@ def parse_snowflake_url(url):
     if len(c) == 1:
         account = c[0]
     else:
+        # for customers
         if path.endswith("snowflakecomputing.com"):
             account = c[0]
             region = c[1] if len(c) == 4 else 'us-west-2'
+        else:
+        # for internal use
+            searchObj = re.search('(.+snowflakecomputing.com):([0-9]+)', path)
```

Comment:
since the port is optional, best to make a regex that accepts strings with and without it. use a tool like https://regex101.com/ and let me know if you need help!